NetCore/Resize.cs

```diff
@@ -127,5 +128,21 @@ public SKSize SkiaBitmapResizeBenchmark()
                 return new SKSize(image.Width, image.Height);
             }
         }
+
+        [Benchmark(Description = "NetVips Resize")]
+        public (int width, int height) NetVipsResize()
+        {
+            // Scaling calculations
+            const double xFactor = (double)Width / ResizedWidth;
+            const double yFactor = (double)Height / ResizedHeight;
+
+            // Make a black image
+            var image = NetVips.Image.Black(Width, Height);
+
+            // Resize
+            image = image.Reduce(xFactor, yFactor, kernel: Enums.Kernel.Linear);
```

Comment:
Isn't this bilinear? If so it's sampling far fewer pixels than the other tests.

Reply:
You're right. I could switch all resize tests to bicubic interpolation (to match `System.Drawing` and `FreeImage`), for example:

```diff
--- NetCore/Resize.cs
+++ NetCore/Resize.cs
@@ -48,7 +48,7 @@
         {
             using (var image = new ImageSharpImage(Width, Height))
             {
-                image.Mutate(i => i.Resize(ResizedWidth, ResizedHeight));
+                image.Mutate(i => i.Resize(ResizedWidth, ResizedHeight, KnownResamplers.Bicubic));
                 return image.Size();
             }
         }
@@ -59,6 +59,9 @@
             var size = new MagickGeometry(ResizedWidth, ResizedHeight);
             using (var image = new MagickImage(MagickColor.FromRgba(0, 0, 0, 0), Width, Height))
             {
+                // 'Catrom' is generally imprecisely known as 'BiCubic' interpolation
+                image.FilterType = FilterType.Catrom;
+
                 image.Resize(size);
                 return size;
             }
@@ -85,7 +88,8 @@
             {
                 Width = ResizedWidth,
                 Height = ResizedHeight,
-                Sharpen = false
+                Sharpen = false,
+                Interpolation = InterpolationSettings.Cubic
             };
 
             using (var pixels = new TestPatternPixelSource(Width, Height, PixelFormats.Bgr24bpp))
@@ -122,7 +126,7 @@
         public SKSize SkiaBitmapResizeBenchmark()
         {
             using (var original = new SKBitmap(Width, Height))
-            using (var resized = original.Resize(new SKImageInfo(ResizedWidth, ResizedHeight), SKFilterQuality.High))
+            using (var resized = original.Resize(new SKImageInfo(ResizedWidth, ResizedHeight), SKFilterQuality.Medium))
             using (var image = SKImage.FromBitmap(resized))
             {
                 return new SKSize(image.Width, image.Height);
@@ -130,7 +134,7 @@
         }
 
         [Benchmark(Description = "NetVips Resize")]
-        public (int width, int height) NetVipsResize()
+        public (int width, int height, byte[] memory) NetVipsResize()
         {
             // Scaling calculations
             const double xFactor = (double)Width / ResizedWidth;
@@ -140,9 +144,13 @@
             var image = NetVips.Image.Black(Width, Height);
 
             // Resize
-            image = image.Reduce(xFactor, yFactor, kernel: Enums.Kernel.Linear);
+            image = image.Resize(1.0 / xFactor, vscale: 1.0 / yFactor, kernel: Enums.Kernel.Cubic);
 
-            return (image.Width, image.Height);
+            // libvips is "lazy" and will not process pixels
+            // until you write to an output file, buffer or memory
+            var memory = image.WriteToMemory();
+
+            return (image.Width, image.Height, memory);
         }
     }
 }
```

I could open a new PR, if you want.

Reply:
Yeah, I think that would be much better. Looking forward to seeing the numbers!

Reply:
PR #17 fixes this.